x-pack/plugins/ml/public/license/check_license.js

```diff
@@ -38,8 +38,16 @@ export function checkLicense(Private, kbnBaseUrl) {
     const message = features.message;
     const exists = _.find(notify._notifs, (item) => item.content === message);
```

Comment:
This duplication check no longer works.
when jumping between pages we now get multiple banners.
<img width="1220" alt="screen shot 2018-07-17 at 07 10 44" src="https://user-images.githubusercontent.com/22172091/42799706-b7dddf5e-8990-11e8-93e4-f95b31a10b41.png">

Reply:
Thanks! I just pushed a fix could you try it again?